Project.toml

```diff
@@ -22,7 +23,6 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Distributions = "0.25"
 EnsembleKalmanProcesses = "0.1"
 JLD2 = "0.4"
-Oceananigans = "0.66, 0.67"
```

Comment:
we don't need that? I guess the repo's main code actually doesn't need it?

Reply:
We're relying on `Manifest.toml` pretty heavily, so the additional check here probably isn't needed.

Reply:
OK. But if we ever want to register the package Julia Registry won't allow us to do so if we don't have compat entries for each dependency. Just saying...